test/kt/esimd_radix_sort.cpp

```diff
@@ -198,41 +178,25 @@ can_run_test(sycl::queue q, KernelParam param)
 int
 main()
 {
-#if TEST_DPCPP_BACKEND_PRESENT
     auto q = TestUtils::get_test_queue();
-    bool run_test = can_run_test<TEST_DATA_TYPE>(q, kernel_parameters);
+    bool run_test = can_run_test<ParamType, TEST_KEY_TYPE>(q, kernel_parameters);
     if (run_test)
     {
-        const std::vector<std::size_t> sizes = {
-            1,       6,         16,      43,        256,           316,           2048,
-            5072,    8192,      14001,   1 << 14,   (1 << 14) + 1, 50000,         67543,
-            100'000, 1 << 17,   179'581, 250'000,   1 << 18,       (1 << 18) + 1, 500'000,
-            888'235, 1'000'000, 1 << 20, 10'000'000};
-
         try
         {
-#if TEST_LONG_RUN
-            for (auto size : sizes)
-            {
-                test_general_cases<TEST_DATA_TYPE, Ascending, TestRadixBits>(q, size, kernel_parameters);
-                test_general_cases<TEST_DATA_TYPE, Descending, TestRadixBits>(q, size, kernel_parameters);
-            }
-            test_small_sizes<TEST_DATA_TYPE, Ascending, TestRadixBits>(q, kernel_parameters);
-#else
-            for (auto size : sizes)
+            for (auto size : sort_sizes)
             {
-                test_usm<TEST_DATA_TYPE, Ascending, TestRadixBits, sycl::usm::alloc::shared>(q, size, kernel_parameters);
-                test_usm<TEST_DATA_TYPE, Descending, TestRadixBits, sycl::usm::alloc::shared>(q, size, kernel_parameters);
+                test_general_cases<TEST_KEY_TYPE, Ascending, TestRadixBits>(q, size, kernel_parameters);
+                test_general_cases<TEST_KEY_TYPE, Descending, TestRadixBits>(q, size, kernel_parameters);
             }
-#endif // TEST_LONG_RUN
+            test_small_sizes<TEST_KEY_TYPE, Ascending, TestRadixBits>(q, kernel_parameters);
         }
         catch (const ::std::exception& exc)
         {
             std::cerr << "Exception: " << exc.what() << std::endl;
-            return EXIT_FAILURE;
+            return 1;
```

Comment:
For that this change required?

Reply:
I've made the change in order not to include `<cstdlib>`. This is a minor change, I do not mind to roll it back and include `<cstdlib>`.

Reply:
See my comments below

Reply:
I have returned it